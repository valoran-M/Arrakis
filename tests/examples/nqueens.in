dune exec arrakis -- ./examples/nqueens.s --no-color --run

