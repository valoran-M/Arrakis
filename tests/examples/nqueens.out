Info: Exiting with code '724'
